### PR TITLE
[Azure Search] Apply branding changes to stable/2019-05-06 and track1/stable/2019-05-06

### PR DESCRIPTION
**Pull request by bleroy**
This applies branding changes from "Azure Search" to "Azure Cognitive Search".

### CODE REVIEW

**Review by arv100kri — CHANGES_REQUESTED** [review]
You'll have to update the references in the following as well

https://github.com/Azure/azure-rest-api-specs/tree/a497484db69d4d468e111972e25f939ef26af2b7/specification/search/data-plane/Microsoft.Azure.Search.Data/stable/2019-05-06
 
https://github.com/Azure/azure-rest-api-specs/blob/a497484db69d4d468e111972e25f939ef26af2b7/specification/search/data-plane/Microsoft.Azure.Search.Data/track1/stable/2019-05-06/searchindex.json

File: specification/search/data-plane/Microsoft.Azure.Search.Service/stable/2019-05-06/searchservice.json [hunk 1]
```diff
@@ -5564,7 +5564,7 @@
       "required": true,
       "type": "string",
       "x-ms-skip-url-encoding": true,
-      "description": "The name of the Azure Search service.",
+      "description": "The name of the Azure Cognitive Search service.",
```
Comment (arv100kri) [inline specification/search/data-plane/Microsoft.Azure.Search.Service/stable/2019-05-06/searchservice.json]: I think we need to have consistency between the searchIndex.json and here...
There we called it "The name of the search service" and here we call it "The name of the Azure Cognitive Search service"....

I prefer what's in searchIndex.json ("The name of the search service") and let's apply that here as well...

File: specification/search/data-plane/Microsoft.Azure.Search.Service/stable/2019-05-06/searchservice.json [hunk 2]
```diff
@@ -5574,7 +5574,7 @@
       "type": "string",
       "default": "search.windows.net",
       "x-ms-skip-url-encoding": true,
-      "description": "The DNS suffix of the Azure Search service. The default is search.windows.net.",
+      "description": "The DNS suffix of the Azure Cognitive Search service. The default is search.windows.net.",
```
Comment (arv100kri) [inline specification/search/data-plane/Microsoft.Azure.Search.Service/stable/2019-05-06/searchservice.json]: "The DNS suffix of the search service...

File: specification/search/data-plane/Microsoft.Azure.Search.Service/track1/stable/2019-05-06/searchservice.json [hunk 3]
```diff
@@ -603,7 +603,7 @@
             "$ref": "./examples/SearchServiceCreateOrUpdateSkillset.json"
           }
         },
-        "description": "Creates a new cognitive skillset in an Azure Search service or updates the skillset if it already exists.",
+        "description": "Creates a new cognitive skillset in a search service or updates the skillset if it already exists.",
```
Comment (arv100kri) [inline specification/search/data-plane/Microsoft.Azure.Search.Service/track1/stable/2019-05-06/searchservice.json]: same comment as in the other file...
"cognitive skillset/skillsets/skills" -> skillset | skillsets | skills

File: specification/search/data-plane/Microsoft.Azure.Search.Service/track1/stable/2019-05-06/searchservice.json [hunk 4]
```diff
@@ -5536,7 +5536,7 @@
       "required": true,
       "type": "string",
       "x-ms-skip-url-encoding": true,
-      "description": "The name of the Azure Search service.",
+      "description": "The name of the Azure Cognitive Search service.",
```
Comment (arv100kri) [inline specification/search/data-plane/Microsoft.Azure.Search.Service/track1/stable/2019-05-06/searchservice.json]: "The name of the search service."

File: specification/search/data-plane/Microsoft.Azure.Search.Service/track1/stable/2019-05-06/searchservice.json [hunk 5]
```diff
@@ -5546,7 +5546,7 @@
       "type": "string",
       "default": "search.windows.net",
       "x-ms-skip-url-encoding": true,
-      "description": "The DNS suffix of the Azure Search service. The default is search.windows.net.",
+      "description": "The DNS suffix of the Azure Cognitive Search service. The default is search.windows.net.",
```
Comment (arv100kri) [inline specification/search/data-plane/Microsoft.Azure.Search.Service/track1/stable/2019-05-06/searchservice.json]: "The DNS suffix of the search service..."

**Review by arv100kri — CHANGES_REQUESTED** [review]
Just some minor comments - once addressed I am ready to sign off :)

**Comment by arv100kri** [comment]
Context -
@sarangan12 - We are rebranding the concept/entity "Azure Search" to "Azure Cognitive Search" and we'd like that to be reflected in our SDK generated doc comments. This achieves that

@HeidiSteen - This is the logic we applied
In places where the term "Azure Search" is used as is (to reference a concept), we replace it with "Azure Cognitive Search".

In places where the term "Azure Search" is followed by a noun ("Azure Search service", "Azure search index", "Azure search indexer etc.,") we simply contract it to "search" + noun or remove "search" altogether as appropriate ("search service", "index", "indexer" etc.,)

File: specification/search/data-plane/Microsoft.Azure.Search.Data/stable/2019-05-06/searchindex.json [hunk 6]
```diff
@@ -2,7 +2,7 @@
   "swagger": "2.0",
   "info": {
     "title": "SearchIndexClient",
-    "description": "Client that can be used to query an Azure Search index and upload, merge, or delete documents.",
+    "description": "Client that can be used to query an Azure Cognitive Search index and upload, merge, or delete documents.",
```
[the remark this answers is not in the document]
Comment (HeidiSteen) [reply specification/search/data-plane/Microsoft.Azure.Search.Data/stable/2019-05-06/searchindex.json]: Agreed -- I've been removing unnecessary references to the service in conceptual docs as well.

**Review by HeidiSteen — APPROVED** [review]
@bleroy Your PR didn't pass validation. You should close and reopen the PR to see if the problem is transitory.  I agree with all of Arvind's comments.

**Comment by arv100kri** [comment]
> @bleroy Your PR didn't pass validation. You should close and reopen the PR to see if the problem is transitory. I agree with all of Arvind's comments.

@HeidiSteen / @sarangan12 / @bleroy - the model validation failures are due to a "ENUM_MISMATCH" which is a known failure mode and we have obtained exceptions from the swagger folks.

**Review by arv100kri — APPROVED** [review]

**Comment by arv100kri** [comment]
@sarangan12 - @brjohnstmsft is on vacation, and I think my sign off is a reasonable substitute.
Could you please merge this pull request at your earliest?

**Comment by sergey-shandar** [comment]
Could you check model validation errors?

**Review by sergey-shandar — APPROVED** [review]

**Pull request merged**